implementation/src/main/java/io/smallrye/health/registry/LivenessHealthRegistry.java

```diff
@@ -0,0 +1,14 @@
+package io.smallrye.health.registry;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Default;
+
+import org.eclipse.microprofile.health.Liveness;
+
+import io.smallrye.health.api.HealthRegistry;
+
+@Liveness
+@Default
```

Comment:
Do we really want to have a `@Default` registry?

Reply:
I didn't want to fail when the qualifier is not used but I can remove this default.

Reply:
IMO it would be better to have the user explicitly qualify the registry and have an unsatisfied injection point error without a qualifier instead of having a default one.